dashboard/test/ui/features/hour_of_code/starwars.feature

```diff
@@ -12,7 +12,7 @@ Feature: Hour of Code 2015 tutorial is completable
     And I press "runButton"
     And I wait to see ".modal"
     Then element "#continue-button" is visible
-    When I close the dialog
+    And I press "continue-button"
```

Comment:
If you want, I think it might also be useful to close the dialog in one of these tests

Reply:
Good call- added a check that the close keeps you in the current page before then intentionally moving forward